scrapi/harvesters/biomedcentral.py

```diff
@@ -86,8 +86,15 @@ def schema(self):
             )
         }
 
-    def harvest(self, days_back=1):
-        search_url = self.URL.format(days_back)
+    def harvest(self, start_date=None, end_date=None):
+
+        start_date = start_date or (date.today() - timedelta(1))
```

Comment:
can you make this consistent across harvesters? the one above it converts these to strings, but this one leaves it as datetime objects.